opendj-server-legacy/src/messages/org/opends/messages/admin_tool.properties

```diff
@@ -2333,6 +2333,7 @@ ERR_CTRL_PANEL_RESETTING_USER_PASSWORD_ERROR_SUMMARY=Error updating user \
  password
 ERR_CTRL_PANEL_RESETTING_USER_PASSWORD_ERROR_DETAILS=An error occurred \
  updating user password.
+INFO_CTRL_PANEL_RESET_PASSWORD_TITLE = Reset password
```

Comment:
We should leave the term "Reset password" / "Password reset" to the password reset feature. I would call the feature simply "Change password".